### PR TITLE
Feature - design - home - view

### PR DESCRIPTION
**Pull request by at-dungduong**

### CODE REVIEW

File: MyApp/Library/Ext/UIImageExt.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  MyApp
+//
+//  Created by Dung Duong C. on 2/22/18.
+//  Copyright © 2018 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import UIKit
+
+extension UIImageView {
+
```
Comment (at-daiho) [inline MyApp/Library/Ext/UIImageExt.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/Model/Schema/Categories.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    static func objectForMapping(map: Map) -> BaseMappable? {
+        do {
+            let realm = try Realm()
+            return realm.object(ofType: self, forMapping: map)
+        } catch {
+            print(error)
+        }
+        return 0 as? BaseMappable
+    }
+
```
Comment (at-daiho) [inline MyApp/Model/Schema/Categories.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/AppDelegate.swift [hunk 3]
```diff
@@ -61,6 +75,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+    
+    func configRealm() {
+        Realm.Configuration.defaultConfiguration = {
+            var config = Realm.Configuration.defaultConfiguration
+            config.deleteRealmIfMigrationNeeded = true
+            return config
+        }()
+    }
+    
+
```
Comment (at-daiho) [inline MyApp/View/AppDelegate.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

File: MyApp/View/AppDelegate.swift [hunk 4]
```diff
@@ -28,10 +29,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window = UIWindow(frame: UIScreen.main.bounds)
         window?.makeKeyAndVisible()
         window?.backgroundColor = .white
+        configRealm()
+        logOut()
         AppDelegate.shared.changeViewNavigation()
         UIApplication.shared.statusBarStyle = .lightContent
         return true
     }
+
```
Comment (at-daiho) [inline MyApp/View/AppDelegate.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/AppDelegate.swift [hunk 5]
```diff
@@ -61,6 +75,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         window?.rootViewController?.present(alert, animated: true, completion: nil)
     }
+
```
Comment (at-daiho) [inline MyApp/View/AppDelegate.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/AppDelegate.swift [hunk 6]
```diff
@@ -61,6 +75,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
     }
+    
+    func configRealm() {
+        Realm.Configuration.defaultConfiguration = {
+            var config = Realm.Configuration.defaultConfiguration
+            config.deleteRealmIfMigrationNeeded = true
+            return config
+        }()
+    }
+
```
Comment (at-daiho) [inline MyApp/View/AppDelegate.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/AppDelegate.swift (hunk 3 above)
Comment (at-daiho) [inline MyApp/View/AppDelegate.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 7
Serious: 0
Warnings: 7

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

File: Demo/Login/Login/ViewController/LoginViewController.swift [hunk 7]
```diff
@@ -86,42 +97,46 @@ class LoginViewController: UIViewController, MVVM.View {
...
         }
+    }
 
-        @IBAction func forgotPasswordTouchInSide(_ sender: Any) {
-            let forgotPasswordViewController = ForgotPasswordViewController()
-            navigationController?.pushViewController(forgotPasswordViewController, animated: true)
-        }
+    @IBAction func createAccountTouchInSide(_ sender: Any) {
+        navigationController?.navigationBar.isHidden = false
+        let registerVC = RegisterViewController(nibName: "RegisterViewController", bundle: nil)
```
Comment (at-tuanmai2) [inline Demo/Login/Login/ViewController/LoginViewController.swift]: defile

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**